### PR TITLE
Zsuzsi userresponse error

### PR DESCRIPTION
**Pull request by vikukzs**

### CODE REVIEW

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java [hunk 1]
```diff
@@ -30,17 +31,20 @@
 public class LoginActivity extends AppCompatActivity {
 
     Button button;
-    EditText editUserName, editPassword;
+    EditText editEmail, editPassword;
 
     TextView textView;
 
-    String username, password;
+    String email, password;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java]: @vikukzs you dont need to store these as class variables

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java [hunk 2]
```diff
@@ -59,27 +63,16 @@ protected void onCreate(Bundle savedInstanceState) {
 
     public void login(View view) {
 
-        username = editUserName.getText().toString();
+        email = editEmail.getText().toString();
         password = editPassword.getText().toString();
-        Call<UserResponse> call = apiService.loginUser(username, password);
+        Call<UserResponse> call = apiService.loginUser(email, password);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java]: @vikukzs just pass to the parameters the getText...

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java (hunk 1 above)
Comment (vikukzs) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java]: done, i think

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java (hunk 2 above)
Comment (vikukzs) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java]: done, i think

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java [hunk 3]
```diff
@@ -59,27 +62,14 @@ protected void onCreate(Bundle savedInstanceState) {
...
-                editor.putString("userName", username);
-                editor.putString("password", password);
-                editor.putBoolean("isLogin", true);
-                editor.apply();
-
-                Toast.makeText(LoginActivity.this, "Saved", Toast.LENGTH_LONG).show();
-
-                Intent i = new Intent(LoginActivity.this, MainActivity.class);
-                startActivity(i);
+                checkCredentialsAndLogIn();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java]: @vikukzs checking if the editTexts are empty should be before sending an "empty" request to the server

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Comment by adamgyulavari** [comment]
@vikukzs also resolve conflicts!

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java (hunk 3 above)
Comment (vikukzs) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java]: i thought it does that...?

Comment (vikukzs) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java]: oh i think i got it.

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**